### ISSUE TITLE
spacing for contact-info went screwy

### ISSUE THREAD

**Issue by heathermcd12** [heathermcd12/dsn1676-term-project#5]
Hi Thomas, 

I know this is a bit late, but I decided to add my behance onto my websites contact area and everything messed up, spacing wise!

![screen shot 2015-04-23 at 8 05 00 pm](https://cloud.githubusercontent.com/assets/6333582/7310085/4db33fdc-e9f4-11e4-85c2-c5f6c65bd4c9.png)

@thomasjbradley

**Comment by thomasjbradley** [comment]
What do you mean "spacing wise"? Like that the cat face is covering the links?

**Comment by heathermcd12** [comment]
Yes! I don't know why its not letting me get to the links, it was working until I added my behance
